React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -290,11 +291,45 @@ - (void)updateProps:(Props::Shared const &)props oldProps:(Props::Shared const &
     self.accessibilityElement.isAccessibilityElement = newViewProps.accessible;
   }
 
+  if (oldViewProps.accessibilityLiveRegion != newViewProps.accessibilityLiveRegion) {
+    NSString *newValueLiveRegion = self.accessibilityLiveRegionAnnouncementType;
+    if (newViewProps.accessibilityLiveRegion == AccessibilityLiveRegion::Polite) {
+      newValueLiveRegion = @"polite";
+    };
+    if (newViewProps.accessibilityLiveRegion == AccessibilityLiveRegion::Assertive) {
+      newValueLiveRegion = @"assertive";
+    };
+    if (newViewProps.accessibilityLiveRegion == AccessibilityLiveRegion::None) {
+      newValueLiveRegion = @"none";
+    };
+
+    for (RCTViewComponentView *subview in self.subviews) {
+      // add logic to over-ride behaviour when subview accessibilityLiveRegion is Polite or Assertive
+      if ([newValueLiveRegion length] != 0) {
+        subview.accessibilityLiveRegionAnnouncementType = newValueLiveRegion;
+      }
+    }
+    self.accessibilityLiveRegionAnnouncementType = newValueLiveRegion;
+  }
+  
+  NSArray *accessibilityLiveRegionAnnouncement = @[@"",@"", @"", @""];
+  NSMutableArray *accessibilityLiveRegionAnnouncementUpdate = [accessibilityLiveRegionAnnouncement mutableCopy];
+  self.accessibilityLiveRegionAnnouncement = @"";
```

Comment:
instead of clearing the announcement, compare if the previous announcement differs from the new announcement to avoid repeating the announcement multiple times.